lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -0,0 +1,427 @@
+            inputElement = getByRole('combobox'),
+            dropdownElement = getByRole('listbox', { hidden: true });
+
+        expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+        expect(dropdownElement).toHaveAttribute('aria-hidden', 'true');
+        await userEvent.type(inputElement, 'f');
+        expect(inputElement).toHaveAttribute('aria-expanded', 'true');
+        expect(dropdownElement).toHaveAttribute('aria-hidden', 'false');
+      });
+
+  it('calls onChange whenver the input\'s onChange event fires', async function() {
+    const onChange = jest.fn(),
+        { getByRole } = rtlRender({ onChange }),
+        inputElement = getByRole('combobox');
+
+    expect(onChange).not.toHaveBeenCalled();
+    await userEvent.type(inputElement, 'a');
+    expect(onChange).toHaveBeenCalledWith('a');
+  });
+
+  it('calls onSearch whenver the input\'s onChange event fires with a value that differs after trimming, ' +
+    'passing the trimmed value', async function() {
+    const onSearch = jest.fn(),
+        { getByRole } = rtlRender({ value: 'foo', onSearch }),
+        inputElement = getByRole('combobox');
+
+    expect(onSearch).not.toHaveBeenCalled();
+    await userEvent.type(inputElement, ' ');
+    expect(onSearch).not.toHaveBeenCalled();
+    await userEvent.type(inputElement, 'f');
+    expect(onSearch).toHaveBeenCalledWith('foof');
+  });
+
+  it('adds the nx-text-input--long class to the div if set', function() {
+    expect(rtlRender().container.querySelector('.nx-combobox')).not.toHaveClass('nx-text-input--long');
+    expect(rtlRender({ className: 'nx-text-input--long' }).container.querySelector('.nx-combobox'))
+        .toHaveClass('nx-text-input--long');
+  });
+
+  it('adds the nx-text-input--short class to the div if set', function() {
+    expect(rtlRender().container.querySelector('.nx-combobox')).not.toHaveClass('nx-text-input--short');
+    expect(rtlRender({ className: 'nx-text-input--short' }).container.querySelector('.nx-combobox'))
+        .toHaveClass('nx-text-input--short');
+  });
+
+  it('passes the disabled prop to the input and buttons', function() {
+    const { getByRole, rerender } = rtlRender(),
+        inputElement = getByRole('combobox');
+
+    expect(inputElement).toHaveProperty('disabled', false);
+
+    rerender(<NxCombobox {...minimalProps} disabled={undefined} />);
+    expect(inputElement).toHaveProperty('disabled', false);
+
+    rerender(<NxCombobox {...minimalProps} disabled={false} />);
+    expect(inputElement).toHaveProperty('disabled', false);
+
+    rerender(<NxCombobox {...minimalProps} matches={[{ id: '1', displayName: 'Foo' }]} disabled={false} />);
+    expect(getByRole('option', { hidden: true })).toHaveProperty('disabled', false);
+
+    rerender(<NxCombobox {...minimalProps} matches={[{ id: '1', displayName: 'Foo' }]} disabled={true} />);
+    expect(getByRole('option', { hidden: true })).toHaveProperty('disabled', true);
```

Comment:
I don't see an assertion that `disabled={true}` gets passed to the input.

Reply:
Updated here 4029d58f1778f46cf53cdedad783b595e4a9da4e.